.github/workflows/release-sync.yml

```diff
@@ -67,4 +67,14 @@ jobs:
       WORKFLOW_ID: "publish-kbcli-scoop.yml"
       VERSION: "${{ needs.update-release-kbcli.outputs.release-version }}"
       APECD_REF: "v0.1.1"
+    secrets: inherit
+
+  release-yum-kbcli:
```

Comment:
@ldming is `goreleaser` WIP?

Reply:
Sorry, can you tell me what goreleaser WIP means?